docs/src/adding_new_equations/nonconservative_advection.md

```diff
@@ -57,40 +51,24 @@ end
 # We use nonconservative terms
 have_nonconservative_terms(::NonconservativeLinearAdvectionEquation) = Val(true)
 
-# OBS! This is scaled by 1/2 because it will cancel later with the factor of 2
-# the flux differencing volume integral
-function calcflux_twopoint_nonconservative!(f1, u, element,
-                                                  equations::NonconservativeLinearAdvectionEquation,
-                                                  dg, cache)
-  for i in eachnode(dg)
-    _, advectionvelocity = get_node_vars(u, equations, dg, i, element)
-
-    for l in eachnode(dg)
-      scalar, _ = get_node_vars(u, equations, dg, l, element)
-      f1[1, l, i] += 0.5 * advectionvelocity * scalar
-    end
-  end
-
-  return nothing
-end
-
-function noncons_interface_flux(u_left, u_right, orientation, mode,
-                                      equations::NonconservativeLinearAdvectionEquation)
-  _, advectionvelocity = u_left
-  scalar, _            = u_right
+function flux_nonconservative(u_mine, u_other, orientation,
+                              equations::NonconservativeLinearAdvectionEquation)
+  _, advectionvelocity = u_mine
+  scalar, _            = u_other
```

Comment:
Could we add short descriptions or comments for the names `u_mine` and `u_other`?

Reply:
I've written something in c124e678. Does that sound reasonable to you?

Reply:
Yes, that looks great - thanks!